CONTRIBUTING.md

```diff
@@ -127,99 +127,77 @@ This will serve the app at [localhost:4141](http://localhost:4141/), and watch f
-make run PROJECT_PATH=<path-to-your-test-project>/new-kedro-project
+make run
 ```
-This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from a real Kedro pipeline located at the project path supplied to the command. From then on, launching the app locally at [localhost:4141](http://localhost:4141) will pull data from the Kedro-Viz server that is running on port 4142. To avoid needing to supply `PROJECT_PATH` every time you call `make run`, you can set it as an environment variable:
+
+This command will launch a Kedro-Viz server at [localhost:4142](http://localhost:4142) and serve data from `demo-project`. From then on, launching the app locally at [localhost:4141](http://localhost:4141) will pull data from the Kedro-Viz server that is running on port 4142.
+
+If you wish to point the server to a different Kedro project then you can do so by altering the `PROJECT_PATH` variable:
 
 ```bash
-export PROJECT_PATH=<path-to-your-test-project>/new-kedro-project 
-make run
+make run PROJECT_PATH=<path-to-your-test-project>/new-kedro-project
 ```
 
 > **Note**: Once the development server is launched at port 4142, the local app will always pull data from that server. To prevent this, you can comment out the proxy setting in `package.json` and restart the dev server at port 4141.
 
 #### Launch the development server with the `SQLiteSessionStore`
 
-Kedro-Viz provides a `SQLiteSessionStore` that users can use in their project to enable experiment tracking functionality. If you want to use this session store with the development server, make sure you don't use a relative path when specifying the store's location. For example, to specify the local `data` directory within a project as the session store's location, configure the project's `settings.py` as follow:
+Kedro-Viz provides a `SQLiteSessionStore` that users can use in their project to enable experiment tracking functionality. If you want to use this session store with the development server, make sure you don't use a relative path when specifying the store's location in `settings.py`. For example, `demo-project` specifies the local `data` directory within a project as the session store's location as follows: 
 ```python
 from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
 SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}
 ```
 
-### Data sources
-
-Kedro-Viz uses a unique identifier to determine the data source from one of several available sources. You can configure this by appending a query string to the URL, e.g. `http://localhost:4141/?data=random`. Alternatively, you can set it with an environment variable when starting up the dev server:
+Owing to this coupling between the project settings and Kedro-Viz, if you wish to execute any Kedro commands on `demo-project` (including `kedro run`), you will need to install the Kedro-Viz Python package. To install your local development version of the package, run:
```

Comment:
You are right that there's two options - either install a released version from PyPI (`pip install kedro-viz`) or install your local development version (the command given). However, when you do `make run` it will always use the local development version (since that sets `PYTHONPATH=$(shell pwd)/package`). So for consistency I think it's best to only use the local development version for the session store as well - otherwise your project is using a different version of kedro-viz from the what the backend server uses. I will clarify this 🙂

Reply:
sounds great - yea let's just scope this to only mention the installation of the local development version ( which is what this section is mainly about anyways). 👍